build/common.sh

```diff
@@ -502,6 +502,7 @@ function kube::build::ensure_data_container() {
       --volume /usr/local/go/pkg/darwin_386_cgo
       --volume /usr/local/go/pkg/windows_amd64_cgo
       --volume /usr/local/go/pkg/windows_386_cgo
+      --volume /godeps
```

Comment:
per [this comment](https://github.com/kubernetes/kubernetes/blob/5c03d3759d7a9065b0b67b1cfb4bc8ee095cd639/build/common.sh#L40-L45) I suspect we might want to bump the tag in `build/build-image/VERSION`.